report/sections/Chapter 11 - Reflection/Project-management.tex

```diff
@@ -0,0 +1,45 @@
+
+While this tool offers a lot of benefits for developers in terms of project management, the tool was unfamiliar to all of the group members, as no member had any experience with the tool. The lack of experience became a hurdle within a few weeks of the project starting. This was due to several factors, including an unwillingness to understand the tool, but the primary issue that we were also learning other tools, frameworks and even a new programming language. All of which were more important to the project and therefore learning Github Projects became a less of a priority. To the point that we completely stopped using it, despite its promises.
+This led us to revert back to tools that we had previously used.
+
+\subsubsection{Notion}
+Notion is a platform that allows users organize resources and manage projects in the same space. The platform allows you to customize the layout of your work space in a way that best suits the team. One particular feature of Notion that we used in this project was the Kanban board. In the Kanban board we defined four categories, similar to what we had in Github Projects, in which we could place work items:
+
+\begin{itemize}
+    \item Not started.
+    \item In progress.
+    \item Review required.
+    \item Done.
+\end{itemize}
+
+While Notion provides fewer features in terms of integration with Github, the tool was familiar to us and provided the functionality that we needed to manage the project.
+Once we switched to Notion, planning and updating the tasks became a more active part of our daily work. This led to better productivity as we were now able to better maintain an overview of what each member of the group was doing and and how far they were with each task.
+
+\section{Agile development approach} \label{sec:agile-dev}
+Irrespective of which tools we used, the goal from the beginning of the project was to use a SCRUM-like approach to project management. 
+Generally, SCRUM is a framework that helps teams and organizations generate value through flexible solutions to complex problems. SCRUM is an iterative and incremental approach to project work where the project is broken up into sprints. A sprint is timeframe, usually 1-4 weeks in length, in which the the teams tries to complete the work that has been selected from a product backlog.
+
+In the spirit of the SCRUM mindset we only adopted the specific things that we felt were necessary. For our project we did not have a SCRUM master or product owner. Instead we collectively created tasks based on features that we wanted to implement. 
```

Comment:
```suggestion
We only adopted the specific things from Scrum that we felt were necessary. For our project we did not have a Scrum master or product owner. Instead, we collectively created tasks based on features that we deem necessary to implement.
In traditional Scrum, you would have product owners that talk to customers about which features are important to them\todo{cite Scrum by Jeff Sutherland}. Since we are both stakeholders and developers, we found that acting as a council is more effective than delegating the role to a single member.
```
Reasoning:
- can't say 'we wanted to'... it's a bit weak of a reason, and we have a better one
- explaining our process for prioritization is important IMO. It's a huge part of project management.